### PR TITLE
bru: add page

### PR DESCRIPTION
**Pull request by nicokosi**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- Version of the command being documented: `1.8.0`

Documentation: https://docs.usebruno.com 📖
The team: https://www.usebruno.com/about 🧑‍💻+🐶

Tested with:

	bru --version
	1.8.0

I dit not mention the `bru run folder` command because I could not make it work. 🤷

### CODE REVIEW

File: pages/common/bru.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+`bru run --env {{environment_name}}`
+
+- Run requests using an environment with a variable:
+
+`bru run --env {{environment_name}} --env-var {{variable_name}}={{variable_value}}`
+
+- Run request and collect the results in an output file:
+
+`bru run --output {{output_file}}.json`
```
Comment (gutjuri) [inline pages/common/bru.md:24]:
```suggestion
`bru run --output {{path/to/output.json}}`
```

File: pages/common/bru.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+- Show help:
```
Comment (gutjuri) [inline pages/common/bru.md:26]:
```suggestion
- Display help:
```

This is a standardised phrase in tldr pages :D

File: pages/common/bru.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+> CLI for Bruno, an Opensource IDE for exploring and testing APIs.
+> More information: <https://docs.usebruno.com/cli/overview.html>.
+
+- Run all request files from the current directory:
+
+`bru run`
+
+- Run a single request from the current directory by specifying its filename:
+
+`bru run {{*.bru}}`
```
Comment (gutjuri) [inline pages/common/bru.md:12]:
```suggestion
`bru run {{path/to/file.bru}}`
```

Does it have to be from the current directory? If yes, we should do:

```suggestion
`bru run {{file.bru}}`
```

**Review by gutjuri — COMMENTED** [review]
Thank you! I have some suggestions:

**Review by gutjuri — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Review by blueskyson — APPROVED** [review]

**Pull request merged**